src/main/topLevelPages/myDashboard/DashboardMenu/DashboardMenu.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormControlLabel, Switch } from '@mui/material';
+import PageContentBlock from '../../../../core/ui/content/PageContentBlock';
+import { DashboardMenuProps } from './DashboardMenuProps';
+import { useDashboardContext } from '../DashboardContext';
+
+/* TODO: extrat in dialogs */
+/*
+  <TipsAndTricks />
+  <InnovationLibraryBlock />
+  <NewMembershipsBlock hiddenIfEmpty />
+  <MyLatestContributions />
+  <RecentForumMessages />
+*/
+
+const DashboardMenu = ({ compact = false }: DashboardMenuProps) => {
+  const context = useDashboardContext();
+
+  const changeView = (event: React.ChangeEvent<HTMLInputElement>) => {
+    context.setActivityEnebled(event.target.checked);
+  };
+
+  return (
+    <PageContentBlock>
+      Menu
+      {!compact && (
+        <FormControlLabel
+          label="Activity View"
```

Comment:
Translations or we will add them later?

Reply:
This is just a layout setpup, the details will be added later with the menu implementation. I might have to change the control entirely (looking into the design).